framework/sysdeps/windows/meson.build

```diff
@@ -22,3 +22,7 @@ framework_files += \
         '../generic/msr.c',
         '../generic/physicaladdress.c',
     )
+
+if run_command([shell, '-c', 'echo $ENABLE_WINE_WORKAROUNDS'], capture: true).stdout().strip() == '1'
+    default_cpp_flags += [ '-DENABLE_WINE_WORKAROUNDS', ]
```

Comment:
maybe we need to start using `add_project_arguments()` for this, like grown-ups, but that's a separate story.

Reply:
I thought of that, but `-Dsomething` requires adding something to meson_options.txt and I didn't want this option to be documented.